feature-libs/organization/account-summary/root/account-summary-root.module.ts

```diff
@@ -0,0 +1,50 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import {
+  AuthGuard,
+  CmsConfig,
+  provideDefaultConfig,
+  provideDefaultConfigFactory,
+} from '@spartacus/core';
+import { AdminGuard } from '@spartacus/organization/administration/core';
+import { ORGANIZATION_ADMINISTRATION_FEATURE } from '@spartacus/organization/administration/root';
+import { CmsPageGuard, PageLayoutComponent } from '@spartacus/storefront';
+import { defaultAccountSummaryRoutingConfig } from './config';
+import { ORGANIZATION_ACCOUNT_SUMMARY_FEATURE } from './feature-name';
+
+// TODO: Inline this factory when we start releasing Ivy compiled libraries
+export function defaultAccountSummaryComponentsConfig(): CmsConfig {
+  const config: CmsConfig = {
+    featureModules: {
+      [ORGANIZATION_ACCOUNT_SUMMARY_FEATURE]: {
```

Comment:
Please take a look of `CheckoutB2BRootModule`. `Checkout-B2B` extends the `base Checkout`. It is very similar to `Account Summary`, which actually extends `administration`. Please also understand the concept of wrapper module. (Take a look of `CheckoutFeatureModule`).